docs/.vuepress/config.js

```diff
@@ -148,7 +148,6 @@ module.exports = {
             ['/guide/supported-browsers', 'Supported browsers'],
             ['/guide/dependencies', 'Dependencies'],
             ['/guide/licensing', 'Licensing'],
-            ['/guide/changelog', 'Changelog'],
```

Comment:
Please remove changelog.md file as well to avoid confusion which one we should keep updated

Reply:
@wojciechczerniak - yup, I removed it already in: 21f936a

Reply:
It's not on the list of changes? 🤔

Reply:
In 21f936a, I can see this: 
<img width="1268" alt="Screen Shot 2021-07-15 at 12 19 09" src="https://user-images.githubusercontent.com/29247866/125772500-b06b9d72-08d9-4c50-8c16-b64e3cc3d9d4.png">